Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -1082,7 +1081,6 @@ static inline bool shouldAddContentKeyRecipients()
         if (videoRenderer->isUsingDecompressionSession())
             return;
 
-        renderer = videoRenderer->renderer();
 #if HAVE(AVSAMPLEBUFFERDISPLAYLAYER_READYFORDISPLAY)
```

Comment:
Can we simplify HAVE_AVSAMPLEBUFFERDISPLAYLAYER_READYFORDISPLAY to PLATFORM(COCOA)?

Also can we simplify this method?
It seems that all is left should be:
```
if (RefPtr videoRenderer = m_videoRenderer)
    videoRenderer->enqueueSample(sample, minimumUpcomingTime);
```

We could then close rdar 117934497

Reply:
don't even need that, this is all in cocoa only code